posthog/api/plugin.py

```diff
@@ -27,15 +28,36 @@
     parse_url,
     reload_plugins_on_workers,
 )
+from posthog.plugins.access import can_root_plugins_via_api
 from posthog.plugins.utils import load_json_file
 from posthog.utils import is_plugin_server_alive
 
 # Keep this in sync with: frontend/scenes/plugins/utils.ts
 SECRET_FIELD_VALUE = "**************** POSTHOG SECRET FIELD ****************"
 
 
+class PluginsAccessLevelPermission(BasePermission):
+    message = "Your organization's plugin access level is insufficient."
+
+    def has_permission(self, request, view) -> bool:
+        min_level = (
+            Organization.PluginsAccessLevel.CONFIG
+            if request.method in SAFE_METHODS
```

Comment:
Q: Where does SAFE_METHODS come from?

Reply:
It's a DRF thing, basically `['GET', 'HEAD', 'OPTIONS']`.